pkg/agent/cniserver/ipam/antrea_ipam_controller.go

```diff
@@ -135,6 +135,7 @@ func (c *AntreaIPAMController) getIPPoolsByPod(namespace, name string) ([]string
 		ipPools, _ := c.ipPoolInformer.Informer().GetIndexer().ByIndex(podIndex, k8s.NamespacedName(namespace, name))
 		for _, item := range ipPools {
 			ipPool := item.(*crdv1a2.IPPool)
+			klog.V(2).InfoS("Found", "IPPool", ipPool)
```

Comment:
I'd suggest to expand this log to include a hint that this is an edge case (otherwise log reader might be surprised this log is not usually present)

Reply:
Thanks. This is a debug log to analyze https://github.com/antrea-io/antrea/issues/3384 , thus we can remove it, the fix is in https://github.com/antrea-io/antrea/pull/3314